pkgs/applications/display-managers/sddm/default.nix

```diff
@@ -93,6 +88,4 @@ stdenv.mkDerivation {
 
     runHook postInstall
   '';
-
-  inherit (unwrapped) meta;
```

Comment:
This inherits SDDM's `meta` attribute so that it shows description, maintainer and other meta-information in `nix-env` and related tools.